CIScripts/Test/Utils/RegisterDefaultResorces.ps1

```diff
@@ -0,0 +1,14 @@
+. ./ContrailUtils.ps1
+
+function Register-DefaultResourcesInContrail {
+    Param (
+        [Parameter(Mandatory = $true)] [string] $ContrailIP
+    )
+
+    $ContrailUrl = "http://" + $ContrailIP + ":8082"
+    $AuthUrl = "http://" + $ContrailIP + ":5000/v2.0"
+    $AuthToken = Get-AccessTokenFromKeystone -AuthUrl $AuthUrl -TenantName "admin" -Username "admin" -Password "c0ntrail123"
```

Comment:
Maybe move all this constants to `Register-DefaultResourcesInContrail` function parameters
MJ & DS